wpsc-core/wpsc-includes.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-if ( defined( 'WPEC_LOAD_DEPRECATED' ) && WPEC_LOAD_DEPRECATED ) {
+if ( defined( 'WPEC_LOAD_DEPRECATED' ) ) {
```

Comment:
not sure why the cleanup rebase/cherry-pick/merge picked it up here ,   but WPEC_LOAD_DEPRECATED could be defined and be false